.github/workflows/semgrep-web.yml

```diff
@@ -0,0 +1,26 @@
+name: Semgrep Web Security Scan
+on:
+  pull_request:
+
+jobs:
+  semgrep-scan:
+    runs-on: ubuntu-latest
+    container:
+      image: returntocorp/semgrep
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v3
+      - name: Run Semgrep
+        env:
+          SEMGREP_RULES: p/sql-injection p/xss p/cwe-top-25 p/owasp-top-ten
+          SEMGREP_SEND_METRICS: off
+        run: |
+          semgrep ci \
+            --junit-xml \
+            -o reports/semgrep.xml
```

Comment:
Don't we need some exclusion folder for this?

Reply:
Since we didn't employ any test framework or else, we can modify it then. What do you think? 🤔

Reply:
Agreed on that 👍